GeneratorLib/CodeGenerator.cs

```diff
@@ -96,33 +97,100 @@ private void EvaluateInheritance(Schema schema)
                 }
             }
 
-            if (schema.Extends == null) return;
+            if (schema.AllOf == null) return;
 
-            // var baseSchema = FileSchemas[schema.Extends.Name];
-            // if (baseSchema.Type.Length == 1 && baseSchema.Type[0].Name == "object") return;
+            foreach (var typeRef in schema.AllOf)
+            {
+                var baseType = FileSchemas[typeRef.Name];
 
-            var baseType = FileSchemas[schema.Extends.Name];
+                if (schema.Properties != null && baseType.Properties != null)
+                {
+                    foreach (var property in baseType.Properties)
+                    {
+                        if (schema.Properties.TryGetValue(property.Key, out Schema value))
```

Comment:
That was unintentional. I just ran some automatic clean up in VS. If you would prefer a lower version of VS as the minimum required, I can make that change.